.github/workflows/test-plugin.yml

```diff
@@ -83,3 +83,76 @@ jobs:
     - name: Run CPU backend examples
       working-directory: triton_shared/python/examples
       run: pytest .
+
+
+  build_and_test_triton_shared_arm:
+    runs-on: ["self-hosted", "1ES.Pool=triton-shared-github-arm"]
+
+    steps:
+    - name: Check Architecture
+      run: uname -a
+
+    - name: Force Failure
+      if: ${{ inputs.force-failure }}
+      run: exit 1
+
+
+    - name: Checkout Triton-Shared
+      uses: actions/checkout@v4
+      with:
+        ref: ${{ inputs.triton-shared-ref }}
+        path: triton_shared
+        submodules: recursive
+
+    - name: Clear Triton Cache
+      run: |
+        rm -rf ~/.triton
+        
+    - name: Modify Instance
+      run: |
+        sudo apt-get update
+        sudo apt-get install -y build-essential ninja-build python3-pip libc6
+    - name: Update PATH
+      run: |
+        echo "PATH=${HOME}/.local/bin:${PATH}" >> "${GITHUB_ENV}"
+    - name: Check pre-commit
+      working-directory: triton_shared/triton
+      run: |
+        python3 -m pip install --upgrade pre-commit
+        python3 -m pre_commit run --all-files --verbose
+    - name: Edit setup for arm64
+      working-directory: triton_shared/triton
+      run: |
+        sed -i 's/arch = {"x86_64": "64", "arm64": "aarch64"}/arch = {"x86_64": "64", "arm64": "aarch64", "aarch64":"aarch64"}/' "python/setup.py"
```

Comment:
Put up a PR for Triton to change setup.py

Reply:
I believe I already did and it's merged see https://github.com/openai/triton/pull/2999

Not sure if we are still using the custom branch of triton, if we're not then I can delete this line